src/GitPrompt.ps1

```diff
@@ -100,6 +100,7 @@ $global:GitPromptSettings = [pscustomobject]@{
     DefaultPromptSuffix                         = '$(''>'' * ($nestedPromptLevel + 1)) '
     DefaultPromptDebugSuffix                    = ' [DBG]$(''>'' * ($nestedPromptLevel + 1)) '
     DefaultPromptEnableTiming                   = $false
+    DefaultPromptAbbreviateHomeDirectory        = $true

```

Comment:
I probably sound like a broken record, but let's default to `$false` for folks who are used to posh-git uncustomized.

Bonus points if you find a place to mention this new setting in the README.